docs/features/customisations.md

```diff
@@ -0,0 +1,15 @@
+# Customisations
+
+Invenio ILS can be tailored to meet the specific needs of different libraries.
+
+## Interface Customisation
+
+The user interface can be customized to reflect the library's branding and aesthetic preferences, enhancing the overall user experience.
+
+An example of the [CERN Library Catalogue](https://catalogue.library.cern) which is built using Invenio ILS!
```

Comment:
I am a bit hesitating if we should link to CERN instance, I am curious what the others think though

Reply:
I would say why not? What are your thoughts?